### DISCUSSION TITLE
select local obj or 3d asset - drag and drop web app.

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#179] [Feature Requests] [answered]
I'm trying to make a web app where a user can drag on drop a (local) 3d model into the scene.
I've done an app where a user can drop or select an image from their local drive, would love the same with 3d models.

It would also be cool if we could do the same for fonts, so a user can select a local font and bring it into a scene to play with.
A lot of typographers would appreciate something like this.
a little testing app for example

**Comment by steam0r** [comment, marked as answer]
hey,

we made some adjustments to this op on dev:

https://cables.gl/op/Ops.Sidebar.LocalFileToDataUrl

this should now play along nicely with GltfScene and FontFile. Please let me know if that helps your cause!

**Reply by marioepsley** [reply]
Thank you.
